src/net.cpp

```diff
@@ -3215,6 +3216,29 @@ void CNode::DisconnectIfBanned()
     }
 }
 
+void CNode::UpdateNodeConfig()
+{
+    skipChecksum = (xVersion.as_u64c(XVer::BU_MSG_IGNORE_CHECKSUM) == 1);
+    if (addrFromPort == 0)
+    {
+        addrFromPort = xVersion.as_u64c(XVer::BU_LISTEN_PORT) & 0xffff;
+    }
+
+    uint64_t num = xVersion.as_u64c(XVer::BU_MEMPOOL_ANCESTOR_COUNT_LIMIT);
+    if (num)
+        nLimitAncestorCount = num; // num == 0 means the field was not provided.
+    num = xVersion.as_u64c(XVer::BU_MEMPOOL_ANCESTOR_SIZE_LIMIT);
+    if (num)
+        nLimitAncestorSize = num;
+
+    xVersion.as_u64c(XVer::BU_MEMPOOL_DESCENDANT_COUNT_LIMIT);
```

Comment:
Should like 3234 make an assignment to `num`?

Reply:
fixed